src/app/sections/send-coins/send-coins.component.scss

```diff
@@ -0,0 +1,16 @@
+@import "../../styles/base.scss";
+
+#send-coin-sect {
+  height: auto; /* Overrides the 500px height that the parallax class applies */
+}
+
+#paragraph-container {
+  color: $white;
+  p {
```

Comment:
i dont think we need this. just let it be the regular paragraph paddings / margins. remove